docs/source/contributing/index.md

```diff
@@ -69,3 +66,11 @@ Make sure you add and commit the generated files in `http-examples`.
 
 conventions
 ```
+
+## Code Formatting and Testing
+
+To ensure consistent code formatting, it's recommended to have the "black" tool installed locally. You can install it by running the "make" command inside the "plone.restapi" folder. Once installed, you can use "black" to automatically format the code.
+
+The project has a set of dependencies required for its proper functioning. You can install these dependencies by running the "make" command, which will handle the installation process for you.
```

Comment:
Let's move this sentence to the very first narrative text in this chapter. Then we don't need to repeat `make`.

```suggestion
```